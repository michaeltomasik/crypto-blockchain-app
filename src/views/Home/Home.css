.Home {
  height: 94vh;
  background: url('https://storage.needpix.com/rsynced_images/bitcoin-3227442_1280.jpg') no-repeat top left fixed;
  background-size: cover;
  padding: 15px;
}

.Home-btc-address {
  cursor: pointer;
}