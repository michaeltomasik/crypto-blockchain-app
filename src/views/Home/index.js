import React from 'react';
import {CopyToClipboard} from 'react-copy-to-clipboard';

import './Home.css';

const exampleBTCAddress = [
  '19wJuNeV2Hd4WTxa8vDrvV3StSdMzoyPi9',
  '3EaWcuNXQrW2JTGy5Cotm4pftovW8UfR2o',
  '3H5gRBEowwSqr3ckY2MKwLsn64KdpU2gQg'
];

const Home = () => (
  <div className="Home">
    <h3>Click to Copy BTC Addresses</h3>
    {exampleBTCAddress.map(address => 
      <p className="Home-btc-address">
        <CopyToClipboard text={address}>
          <div>- {address}</div>
        </CopyToClipboard>
      </p>)}
  </div>
);

export default Home;
