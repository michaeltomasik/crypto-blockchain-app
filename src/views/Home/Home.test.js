import React from 'react';
import { mount } from 'enzyme';

import Home from './index';

it('renders home page', () => {
  const wrapper = mount(
    <Home />
  );

  expect(wrapper.html()).toMatchSnapshot();
});
