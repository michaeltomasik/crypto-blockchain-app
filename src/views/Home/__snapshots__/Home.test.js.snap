// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders home page 1`] = `"<div class=\\"Home\\"><h3>Click to Copy BTC Addresses</h3><div class=\\"Home-btc-address\\"><div>- 19wJuNeV2Hd4WTxa8vDrvV3StSdMzoyPi9</div></div><div class=\\"Home-btc-address\\"><div>- 3EaWcuNXQrW2JTGy5Cotm4pftovW8UfR2o</div></div><div class=\\"Home-btc-address\\"><div>- 3H5gRBEowwSqr3ckY2MKwLsn64KdpU2gQg</div></div></div>"`;
